### DISCUSSION TITLE
建议加一个自动将BDMV文件转换为ISO镜像文件的功能

### DISCUSSION THREAD

**Discussion by jinws1993** [jxxghp/MoviePilot#2858] [建议]
由于下载的bdmv文件有上千个文件，不利于后面整理，希望增加一个识别到蓝光文件后可以自动将蓝光的BDMV文件打包成ISO镜像文件。

**Comment by cng10001** [comment]
![PixPin_2024-10-22_13-10-39](https://github.com/user-attachments/assets/088b4a63-f80d-4641-af2e-b7f69586fa99)
试试这个

**Reply by tellok** [reply]
我咋没搜到这个插件，需要新版本的moviepilot？

**Reply by cng10001** [reply]
把wiki里的插件市场都加进去

**Reply by jinws1993** [reply]
这个插件咋用？没发现使用说明

**Reply by tellok** [reply]
应该是把蓝光原盘的文件，dump成一个单独的mkv之类的吧

**Reply by tellok** [reply]
> 把wiki里的插件市场都加进去

能问一下具体是哪个第三方插件目录吗，wiki里写的很多啊

**Reply by cng10001** [reply]
你看图片上有作者名字的

**Reply by tellok** [reply]
找到了，是这个https://github.com/HankunYu/MoviePilot-Plugins
